Exporter/src/main/resources/queries.xml

```diff
@@ -209,4 +209,7 @@
     select accNo from Section sec where sec.type = 'Publication' and sec.submission_id = :submissionId
   </entry>
 
+  <entry key="submissionStatsQuery">
+    SELECT value from SubmissionStat where accNo = :accNo and type = :type
```

Comment:
I think we should make this query not results safe instead of handling it in code. I think COALESCE will make the trick. 

```
SELECT COALESCE(value, 0) from SubmissionStat where accNo = :accNo and type = :type

```

Reply:
Nice catch. What actually needs to be used here is `IFNULL` method:

https://stackoverflow.com/a/12450344/2934857